openedx/adg/lms/webinars/tests/test_views.py

```diff
@@ -49,7 +49,7 @@ def test_webinar_description_view_valid_pk(mock_select_template, user_client):
     Test webinar description with valid pk
     """
     _, client = user_client
-    webinar = WebinarFactory(status=Webinar.CANCELLED)
+    webinar = WebinarFactory(is_cancelled=False)
```

Comment:
shouldn't it be `is_cancelled=True` since the deleted line is `status=Webinar.CANCELLED`?

Reply:
It's corrected for both these states. Here we are creating a webinar and verifying if it's detailed page is called with the webinar primary key(webinar id). Test is passing for both states.